community-roadmap/README.md

```diff
@@ -0,0 +1,13 @@
+# Community Roadmap for Zendesk Help Center
+
+This project provides a public roadmap feature in Zendesk Help Center, where the product roadmap items are driven by community posts.
+
+## Structure
+
+The project is divided into two major modules:
+
+The `backend` module is a REST API service built for hosting outside of Zendesk.
+
+The `frontend` module is the  client of the REST API, built to run inside Zendesk Help Center, for instance as a Custom Page.
```

Comment:
Unless I'm mistaken, there isn't any other way for a user to embed an app into a help center.
```suggestion
The `frontend` module is the  client of the REST API, built to run inside a help center as a custom page.
```